packages/nimbus/src/components/loading-spinner/loading-spinner.mdx

```diff
@@ -9,84 +9,217 @@ menu:
   - LoadingSpinner
 tags:
   - component
+
 ---
 
-# LoadingSpinner
+# Loading spinner
```

Comment:
@FilPob @ellelevy I made an executive decision to title this as `Loading spinner` simply because it's called `LoadingSpinner` everywhere else in the codebase, and I fear that consumers may look for components called `ProgressBar` during the migration. Later on in this document, we do also inform consumers of our intentions:

```
**Currently we support the loading spinner, later we will be reworking to provide a classic progress bar.**
```


We can always change it later - however, let me know if you find this to be problematic and I will revert it.

Reply:
Yeah I am fine with having 2 separate components, a `loading spinner` and the `progress bar` for the time being. But in most design systems these are done as one component that can be toggled between a spinner and a progress bar